fontconfig/src/lib.rs

```diff
@@ -278,7 +279,7 @@ impl<'fc> Pattern<'fc> {
 impl<'fc> std::fmt::Debug for Pattern<'fc> {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         let fcstr = unsafe { sys::FcNameUnparse(self.pat) };
```

Comment:
To be consistent, does this one need to be cast as well?

Reply:
This is an `FcPattern` so I _think_ it's ok here?